quilt/tools/command.py

```diff
@@ -171,14 +171,28 @@ def logout():
     else:
         print("Already logged out.")
 
+def generate(directory):
+    """
+    Generate a build-file for quilt build from a directory of
+    source files.
+    """
+    try:
+        buildfilepath = generate_build_file(directory)
+    except BuildException as builderror:
+        raise CommandException(str(builderror))
+
+    print("Generated build-file %s." % (buildfilepath))
+
 def build(package, path, directory=None):
```

Comment:
this line blows up `TypeError: build() got an unexpected keyword argument 'auto'`